### ISSUE TITLE
Big icon sidebar menu 

### ISSUE THREAD

**Issue by iqfareez** [scalar/scalar#1097]
### What happens?

Sidebar menu icon appear to be big

![image](https://github.com/scalar/scalar/assets/60868965/504dc2ae-c98b-4435-94ab-73b4e93f5700)

Also, the open menu text is shown where it shouldn't be.

![image](https://github.com/scalar/scalar/assets/60868965/8a30fbe4-9af8-4bac-bd1e-b013b97b5fcb)

The behavior only appears on mobile view

https://github.com/scalar/scalar/assets/60868965/e589b98d-9f1c-4ef6-81d6-c3ae79abc604

### What did you expect to happen?

The menu icon correct size

![image](https://github.com/scalar/scalar/assets/60868965/608794ce-cf8f-4e95-8a92-4775fa9b84ab)

Workaround for now, pin @scalar/api-reference to version `1.17.8` for now.

### How can we reproduce the issue?

package.json (Only the important part I think)

```json
"dependencies": {
    "@scalar/api-reference": "1.17.13",
    ...
    "veaury": "^2.3.12"
  },
```

Framework: Next JS

I used the mount to React component method

```tsx
import { ApiReference as VueComponent } from '@scalar/api-reference'
import { applyVueInReact } from 'veaury'

const ApiReference = applyVueInReact(VueComponent)

function App() {
    const configuration = {
        spec: {
            url: 'https://raw.githubusercontent.com/mptwaktusolat/open-api-spec/main/mpt-server-api.yaml'
        },
        metaData: {
            title: 'API References | Waktu Solat API',
            description: 'Easily integrate Waktu Solat API into your applications for accurate prayer times and zone detection',
            ogDescription: 'Easily integrate Waktu Solat API into your applications for accurate prayer times and zone detection',
            ogTitle: 'API References | Waktu Solat API',
            ogImage: 'https://mpt-server.vercel.app/metaimage.png',
            twitterCard: 'summary_large_image',
          }
    };
    return (
        <>
            {/* @ts-ignore */}
            <ApiReference configuration={configuration} />
        </>
    )
}

export default App
```

**Comment by marclave** [comment]
@iqfareez so sorry about this! we are investigating this right now ✨ will update with a fix soon

**Comment by hanspagel** [comment]
Bummer, we’re on it!

**Comment by marclave** [comment]
hi @iqfareez this is fixed in `@scalar/api-reference@1.17.14`

let me know if this works for you ✨

**Comment by iqfareez** [comment]
Thank you for the prompt response. Yes! the new version fixed this issue. 😄 👍
